### ISSUE TITLE
Support for Ed448

### ISSUE THREAD

**Issue by kaduk** [openssl/openssl#5049]
#3585 added support for Ed25519, so moving the last bit of #487 out into its own issue so that it can be closed.  Ed448 support is blocking on a license-compatible implementation of the underlying primitives.

**Comment by mattcaswell** [comment]
We have a licence-compatible implementation available to us. See #4829. Its just waiting on me having the cycles to finish this work off.

**Comment by mattcaswell** [comment]
I am hoping this will be in 1.1.1, so removing the Post 1.1.1 label (at least for now).

**Comment by mattcaswell** [comment]
I have replaced #4829 with #5105.

**Comment by jrchamp** [comment]
@mattcaswell It looks like that PR is closed and merged? I'm not seeing any mention in the 1.1.1 short or full changelog: https://www.openssl.org/news/cl111.txt

**Comment by mattcaswell** [comment]
> It looks like that PR is closed and merged? I'm not seeing any mention in the 1.1.1 short or full changelog

See:
https://github.com/openssl/openssl/blob/master/CHANGES#L39

````
  *) Added support for X448 and Ed448. Heavily based on original work by
     Mike Hamburg.
     [Matt Caswell]
````
